observation/batch_reader.go

```diff
@@ -0,0 +1,36 @@
+package observation
+
+// BatchReader provides an interface to read Observations
+type BatchReader interface {
```

Comment:
It is not currently used. In the original CSV splitter we were using the Kafka sync producer with batch message writes. I need to determine if the async producer is safe to use. If it is then the batching is not required and can be removed.